File: app/src/main/java/com/example/carbonfootprints/Signup.java
package com.example.carbonfootprints;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;


public final class Signup extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.setContentView(R.layout.activity_signup);
        Button signUpButton = findViewById(R.id.signupButtonG);
        signUpButton.setOnClickListener(v -> signUp());    }


    private void signUp() {
        //SWITCH TO DASHBOARD when dashboard is created
        Intent intent = new Intent(this, MainActivity.class);
        this.startActivity(intent);
    }
}

File: app/src/main/java/com/example/carbonfootprints/Signup.kt
package com.example.carbonfootprints

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class Signup : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_signup)
        setListeners()
    }

    private fun setListeners() {
        val loginTextView = findViewById<TextView>(R.id.loginBottomRight)
        val signUpButton = findViewById<Button>(R.id.signUpButton)
        loginTextView.setOnClickListener{
            login()
        }
    }

    private fun login() {
        val intent = Intent(this, Login::class.java);
        startActivity(intent)
    }

}